src/FirstSteps/RPG/HandlingItemException.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+
+namespace FirstSteps.RPG
+{
+    public class HandlingItemException : Exception
```

Comment:
The name of an exception should say more about the reason for throwing it.
For example, when you want to throw an exception when you cannot find a user in a database you can create UserNotFoundException.